# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../habak/imlib.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


Not every system has imlib2 in /usr ...

  - Rene Rebe <rene@exactcode.de>

diff -ur habak-0.2.5/src/Makefile habak-0.2.5-fixed/src/Makefile
--- habak-0.2.5/src/Makefile	2004-01-08 14:52:54.000000000 +0100
+++ habak-0.2.5-fixed/src/Makefile	2006-02-02 12:28:34.000000000 +0100
@@ -1,8 +1,8 @@
 #CFLAGS = -I/usr/X11R6/include -Wall -g -ggdb
 #LDFLAGS = -L/usr/X11R6/lib -lImlib2 -lm -g -ggdb
 
-CFLAGS += -I/usr/X11R6/include -Wall
-LDFLAGS += -L/usr/X11R6/lib -lImlib2 -lm
+CFLAGS += -I/usr/X11R6/include -Wall `imlib2-config --cflags`
+LDFLAGS += -L/usr/X11R6/lib -lm `imlib2-config --libs`
 
 all:	habak
 	ln -f habak ../habak
